### ISSUE TITLE
Fix docstring formatting warnings

### ISSUE THREAD

**Issue by pp-mo** [SciTools/cartopy#978]
Since upgrading to use sphinx-gallery, we seem to have a whole lot of new warnings, relating to problems with the docstring formats of example code.
as in [this example](https://github.com/SciTools/cartopy/pull/969#issuecomment-351031968)

Probably could all be usefully revised to avoid these warnings.

For example, it objects to 
```
    def tile_bbox(self, x, y, z, y0_at_north_pole=True):
        """
        Returns the ``(x0, x1), (y0, y1)`` bounding box for the given x, y, z
        tile position.

        Parameters
        ----------
        x, y, z : int
            The x, y, z tile coordinates in the Google tile numbering system
```
In `cartopy.io`img_tiles.tile_bbox`, error at line 13 which is the "Parameters" line.
The warnings, and html results, imply that it is currently just **skipping** section headings like this one.

**Comment by corinnebosley** [comment]
One of the problems that we were having was the failure of a couple of hyperlinks.  After running a link-check, the error message for one of the broken links looked like this:

```
(line   15) broken    http://www.lfd.uci.edu/~gohlke/pythonlibs/ - HTTPSConnectionPool(host='www.lfd.uci.edu', port=443): Max retries exceeded with url: /~gohlke/pythonlibs/ (Caused by SSLError(SSLError("bad handshake: SysCallError(-1, 'Unexpected EOF')",),))
```

It is likely that this has been caused by Requests removing their support for 3DES in v2.12 (I downgraded my environment to run with v2.11 and did not get the error).

There's not really anything that we can do about this, so I am going to leave the URL in the file and just not render it as a link.  I reckon that's the best compromise here.
